.travis.yml

```diff
@@ -2,22 +2,18 @@ sudo: false
 language: node_js
 node_js:
   - '6'
-  - '5'
-  - '4'
```

Comment:
we are not dropping the support for node4, just node 0.12 and earlier. I removed it from the test cause there were some issues with running the test 3 times in parallel. But maybe we should test on node4 only